tests/integration/devfile/cmd_dev_test.go

```diff
@@ -290,6 +291,28 @@ var _ = Describe("odo dev command tests", func() {
 					Expect(err).ToNot(HaveOccurred())
 				})
 			})
+
+			When("recording telemetry data", func() {
+				BeforeEach(func() {
+					helper.EnableTelemetryDebug()
+					helper.CopyExample(filepath.Join("source", "devfiles", "nodejs", "project"), commonVar.Context)
+					helper.Cmd("odo", "init", "--name", cmpName, "--devfile-path", helper.GetExamplePath("source", "devfiles", "nodejs", "devfile.yaml")).ShouldPass()
+					helper.StartDevMode().Stop()
```

Comment:
This is not correct anymore because `StartDevMode()` return statement has changed.
```suggestion
					Expect(helper.RunDevMode()).To(BeNil())
```